### ISSUE TITLE
Unable to build latest version with yarn

### ISSUE THREAD

**Issue by effofxprime** [ping-pub/explorer#210]
Issue:  Yarn fails to build.
Versions:
```
root@erialos:~/sept-fork-explorer-update# node --version
v16.15.1
root@erialos:~/sept-fork-explorer-update# npm version
y{
  'ping-wallet': '0.1.0',
  npm: '8.11.0',
  node: '16.15.1',
  v8: '9.4.146.24-node.21',
  uv: '1.43.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '93',
  nghttp2: '1.47.0',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '1.1.1o+quic',
  cldr: '40.0',
  icu: '70.1',
  tz: '2021a3',
  unicode: '14.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}
root@erialos:~/sept-fork-explorer-update# yarn --version
3.2.3
```

When I run `yarn && yarn build`, it fails to build with the following output.

```
root@erialos:~/sept-fork-explorer-update# yarn && yarn build
➤ YN0070: Migrating from Yarn 1; automatically enabling the compatibility node-modules linker 👍

➤ YN0000: ┌ Resolution step
➤ YN0061: │ vue-context@npm:6.0.0 is deprecated: No longer maintained
➤ YN0061: │ uuid@npm:3.4.0 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0061: │ core-js@npm:3.22.5 is deprecated: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
➤ YN0061: │ core-js@npm:2.6.12 is deprecated: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
➤ YN0032: │ eccrypto@npm:1.1.6: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ secp256k1@npm:4.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ secp256k1@npm:3.7.1: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ popper.js@npm:1.16.1 is deprecated: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
➤ YN0061: │ querystring@npm:0.2.0 is deprecated: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
➤ YN0061: │ highlight.js@npm:9.18.5 is deprecated: Support has ended for 9.x series. Upgrade to @latest
➤ YN0032: │ nan@npm:2.14.0: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ nan@npm:2.16.0: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ evp_bytestokey@npm:1.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ node-addon-api@npm:2.0.2: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ keccak@npm:3.0.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ stable@npm:0.1.8 is deprecated: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
➤ YN0002: │ @vue/babel-plugin-jsx@npm:1.1.1 doesn't provide @babel/core (p36aab), requested by @babel/plugin-syntax-jsx
➤ YN0002: │ @vue/cli-service@npm:5.0.4 [2fb66] doesn't provide @babel/core (p1b57a), requested by @babel/helper-compilation-targets
➤ YN0002: │ @vue/eslint-config-airbnb@npm:6.0.0 [2fb66] doesn't provide webpack (pa6ebc), requested by eslint-import-resolver-webpack
➤ YN0002: │ @vue/eslint-config-standard@npm:6.1.0 [2fb66] doesn't provide webpack (p11b8d), requested by eslint-import-resolver-webpack
➤ YN0002: │ @vueuse/core@npm:4.0.0 doesn't provide vue (p71f68), requested by vue-demi
➤ YN0002: │ @vueuse/shared@npm:4.0.0 doesn't provide vue (p70418), requested by vue-demi
➤ YN0002: │ bootstrap-vue@npm:2.21.1 doesn't provide jquery (p68a24), requested by bootstrap
➤ YN0002: │ bootstrap-vue@npm:2.21.1 doesn't provide vue (pa8cbc), requested by portal-vue
➤ YN0002: │ ping-wallet@workspace:. doesn't provide @babel/core (p85d5f), requested by @babel/eslint-parser
➤ YN0002: │ ping-wallet@workspace:. doesn't provide @types/leaflet (p82bb3), requested by vue2-leaflet
➤ YN0060: │ ping-wallet@workspace:. provides apexcharts (pd9bff) with version 3.2.0, which doesn't satisfy what vue-apexcharts requests
➤ YN0002: │ ping-wallet@workspace:. doesn't provide css-loader (p3fbb9), requested by vue-loader
➤ YN0060: │ ping-wallet@workspace:. provides eslint (p4ac6c) with version 8.15.0, which doesn't satisfy what @vue/eslint-config-standard and some of its descendants request
➤ YN0002: │ ping-wallet@workspace:. doesn't provide eslint-plugin-node (p9b947), requested by @vue/eslint-config-standard
➤ YN0002: │ ping-wallet@workspace:. doesn't provide eslint-plugin-promise (pd0be4), requested by @vue/eslint-config-standard
➤ YN0060: │ ping-wallet@workspace:. provides eslint-plugin-vue (pe8a54) with version 9.0.1, which doesn't satisfy what @vue/eslint-config-standard requests
➤ YN0002: │ ping-wallet@workspace:. doesn't provide jquery (p45d43), requested by bootstrap
➤ YN0002: │ ping-wallet@workspace:. doesn't provide popper.js (p882e5), requested by bootstrap
➤ YN0002: │ ping-wallet@workspace:. doesn't provide svelte (p6f170), requested by swiper
➤ YN0060: │ ping-wallet@workspace:. provides swiper (p375f6) with version 6.5.1, which doesn't satisfy what vue-awesome-swiper requests
➤ YN0060: │ ping-wallet@workspace:. provides vue (p165c3) with version 2.6.14, which doesn't satisfy what @intlify/vue-i18n-loader requests
➤ YN0002: │ ping-wallet@workspace:. doesn't provide webpack (p84cf9), requested by compression-webpack-plugin
➤ YN0002: │ ping-wallet@workspace:. doesn't provide webpack (pe80da), requested by node-polyfill-webpack-plugin
➤ YN0002: │ ping-wallet@workspace:. doesn't provide webpack (p79ce6), requested by sass-loader
➤ YN0002: │ ping-wallet@workspace:. doesn't provide webpack (p98593), requested by ts-loader
➤ YN0002: │ ping-wallet@workspace:. doesn't provide webpack (p2d89e), requested by vue-loader
➤ YN0002: │ vue-context@npm:6.0.0 doesn't provide vue (p0d58e), requested by vue-clickaway
➤ YN0002: │ vue-slider-component@npm:3.2.11 doesn't provide vue (p5afb8), requested by vue-property-decorator
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 27s 275ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ yargs-parser@npm:20.2.9 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yargs@npm:16.2.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yorkie@npm:2.0.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zrender@npm:5.3.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ typescript@npm:4.6.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: └ Completed in 2m 4s
➤ YN0000: ┌ Link step
➤ YN0007: │ protobufjs@npm:6.11.2 must be built because it never has been before or the last one failed
➤ YN0007: │ protobufjs@npm:6.10.2 must be built because it never has been before or the last one failed
➤ YN0007: │ @tharsis/address-converter@npm:0.1.8 must be built because it never has been before or the last one failed
➤ YN0007: │ @tharsis/eip712@npm:0.2.4 must be built because it never has been before or the last one failed
➤ YN0007: │ @tharsis/proto@npm:0.1.19 must be built because it never has been before or the last one failed
➤ YN0007: │ @tharsis/eip712@npm:0.2.3 must be built because it never has been before or the last one failed
➤ YN0007: │ core-js@npm:2.6.12 must be built because it never has been before or the last one failed
➤ YN0007: │ bootstrap-vue@npm:2.21.1 must be built because it never has been before or the last one failed
➤ YN0007: │ core-js@npm:3.22.5 must be built because it never has been before or the last one failed
➤ YN0007: │ protobufjs-dbx@npm:7.0.0 must be built because it never has been before or the last one failed
➤ YN0007: │ swiper@npm:6.5.1 [2fb66] must be built because it never has been before or the last one failed
➤ YN0007: │ vue-echarts@npm:6.0.3 [2fb66] must be built because it never has been before or the last one failed
➤ YN0007: │ vue-demi@npm:0.12.5 [293a2] must be built because it never has been before or the last one failed
➤ YN0007: │ secp256k1@npm:3.7.1 must be built because it never has been before or the last one failed
➤ YN0007: │ secp256k1@npm:4.0.3 must be built because it never has been before or the last one failed
➤ YN0007: │ highlight.js@npm:9.18.5 must be built because it never has been before or the last one failed
➤ YN0007: │ yorkie@npm:2.0.0 must be built because it never has been before or the last one failed
➤ YN0007: │ keccak@npm:3.0.2 must be built because it never has been before or the last one failed
➤ YN0007: │ @tharsis/transactions@npm:0.2.5 must be built because it never has been before or the last one failed
➤ YN0007: │ eccrypto@npm:1.1.6 must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 1m 1s
➤ YN0000: Done with warnings in 3m 33s

⠦  Building legacy bundle for production...(node:609499) [DEP_WEBPACK_JAVASCRIPT_MODULES_PLUGIN] DeprecationWarning: webpack.JavascriptModulesPlugin has moved to webpack.javascript.JavascriptModulesPlugin
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:609499) [DEP_WEBPACK_LIBRARY_TEMPLATE_PLUGIN] DeprecationWarning: webpack.LibraryTemplatePlugin is deprecated and has been replaced by compilation.outputOptions.library or compilation.addEntry + passing a library option
(node:609499) [DEP_WEBPACK_SINGLE_ENTRY_PLUGIN] DeprecationWarning: SingleEntryPlugin was renamed to EntryPlugin
(node:609499) [DEP_WEBPACK_OPTIONS_DEFAULTER] DeprecationWarning: webpack.WebpackOptionsDefaulter is deprecated and has been replaced by webpack.config.getNormalizedWebpackOptions and webpack.config.applyWebpackOptionsDefaults
(node:609499) [DEP_WEBPACK_AGGRESSIVE_SPLITTING_PLUGIN] DeprecationWarning: AggressiveSplittingPlugin is deprecated in favor of SplitChunksPlugin
(node:609499) [DEP_WEBPACK_COMPILATION_ADDITIONAL_CHUNK_ASSETS] DeprecationWarning: additionalChunkAssets is deprecated (use Compilation.hooks.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)
(node:609499) [DEP_WEBPACK_COMPILATION_AFTER_OPTIMIZE_CHUNK_ASSETS] DeprecationWarning: afterOptimizeChunkAssets is deprecated (use Compilation.hooks.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)
(node:609499) [DEP_WEBPACK_COMPILATION_NORMAL_MODULE_LOADER_HOOK] DeprecationWarning: Compilation.hooks.normalModuleLoader was moved to NormalModule.getCompilationHooks(compilation).loader
(node:609499) [DEP_WEBPACK_COMPILATION_OPTIMIZE_CHUNK_ASSETS] DeprecationWarning: optimizeChunkAssets is deprecated (use Compilation.hooks.processAssets instead and use one of Compilation.PROCESS_ASSETS_STAGE_* as stage option)
(node:609499) [DEP_WEBPACK_CONTEXT_MODULE_FACTORY_ALTERNATIVES] DeprecationWarning: ContextModuleFactory.hooks.alternatives has deprecated in favor of ContextModuleFactory.hooks.alternativeRequests with an additional options argument.
⠋  Building legacy bundle for production...

 ERROR  Failed to compile with 3 errors                                                                                                 7:51:44 AM

 error  in ./src/libs/client/EthereumLedgerSigner.ts

Module parse failed: Unexpected token (29:26)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| }
|
> function toPubkey(keyBytes: Uint8Array): Pubkey {
|   return {
|     type: 'ethermint.crypto.v1.ethsecp256k1.PubKey',

 error  in ./src/libs/client/SigningEthermintClient.ts

Module parse failed: Unexpected token (33:7)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import { defaultMessageAdapter } from './MessageAdapter'
|
> export interface TypedDataField {
|     name: string;
|     type: string;

 error  in ./src/libs/client/SigningKeplrEthermintClient.ts

Module parse failed: Unexpected token (14:10)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| export default class SigningKeplerEthermintClient {
>   private signer
|
|   private client

 ERROR  Error: Build failed with errors.
Error: Build failed with errors.
    at /root/sept-fork-explorer-update/node_modules/@vue/cli-service/lib/commands/build/index.js:207:23
    at /root/sept-fork-explorer-update/node_modules/webpack/lib/webpack.js:148:8
    at /root/sept-fork-explorer-update/node_modules/webpack/lib/HookWebpackError.js:68:3
    at Hook.eval [as callAsync] (eval at create (/root/sept-fork-explorer-update/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/root/sept-fork-explorer-update/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.shutdown (/root/sept-fork-explorer-update/node_modules/webpack/lib/Cache.js:150:23)
    at /root/sept-fork-explorer-update/node_modules/webpack/lib/Compiler.js:1225:15
    at Hook.eval [as callAsync] (eval at create (/root/sept-fork-explorer-update/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/root/sept-fork-explorer-update/node_modules/tapable/lib/Hook.js:18:14)
    at Compiler.close (/root/sept-fork-explorer-update/node_modules/webpack/lib/Compiler.js:1218:23)
    at /root/sept-fork-explorer-update/node_modules/webpack/lib/webpack.js:147:16
    at finalCallback (/root/sept-fork-explorer-update/node_modules/webpack/lib/Compiler.js:441:32)
    at /root/sept-fork-explorer-update/node_modules/webpack/lib/Compiler.js:458:13
    at eval (eval at create (/root/sept-fork-explorer-update/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:29:1)
    at done (/root/sept-fork-explorer-update/node_modules/webpack-bundle-analyzer/lib/BundleAnalyzerPlugin.js:80:9)
    at Hook.eval [as callAsync] (eval at create (/root/sept-fork-explorer-update/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:25:1)

```

Im currently running an older version of ping.pub that I do have working and can currently compile using the nodejs/npm/yarn versions that I have installed.

Is there a specific version of nodejs/npm/yarn that you use/is required?

**Comment by effofxprime** [comment]
I have discovered why I could not build the project.
The issue was in my vue.config.js.  When running yarn && yarn build, there are a few size warnings.  I had added this to the file in prior versions which had worked, but now causes it to fault like above.
```
chainWebpack: config => {
    config.performance
    .maxEntrypointSize(10240000)
    .maxAssetSize(10240000)
    config.optimization.minimize(true);
    config.optimization.splitChunks({
      chunks: 'all',
      minSize: 81390,
      maxSize: 122880,
    });
  },
```
It's possible my formatting for this is off for the new version as well. Since I know where the failure is now, I am going to try and get this to work and then report back one last time.
